core/pom.xml

```diff
@@ -107,6 +107,11 @@
         </dependency>
 
         <dependency>
+            <groupId>org.jooq</groupId>
+            <artifactId>jool</artifactId>
```

Comment:
Cool, we should chat on how you like this. I played with javaslang in francois, and really like the Try model for lambdas with checked exceptions